lego/apps/users/tests/test_models.py

```diff
@@ -364,28 +476,30 @@ def test_penalty_offset_is_calculated_correctly(self, mock_now):
         # This penalty is set to expire the day before the penalty freeze
         # It should not be active
         inactive = Penalty.objects.create(
-            created_at=mock_now().replace(day=1),
+            created_at=mock_now().replace(day=10),
             user=self.test_user,
             reason="inactive",
             weight=1,
             source_event=self.source,
         )
         self.assertEqual(self.test_user.number_of_penalties(), 0)
         self.assertEqual(
-            (inactive.exact_expiration.month, inactive.exact_expiration.day), (12, 21)
+            (inactive.exact_expiration.month, inactive.exact_expiration.day),
+            (12, inactive.created_at.day + settings.PENALTY_DURATION.days),
         )
 
         # This penalty is set to expire the same day as the freeze
         active = Penalty.objects.create(
-            created_at=mock_now().replace(day=2),
+            created_at=mock_now().replace(day=5),
             user=self.test_user,
             reason="active",
             weight=1,
             source_event=self.source,
         )
-        self.assertEqual(self.test_user.number_of_penalties(), 1)
+        # 19 = 11: end of holiday + 10: penalty time - 2: from activation time to start of holiday
```

Comment:
I don't get it :'( could you make this into a calculation of variables as well, such as https://github.com/webkom/lego/pull/3349/files#diff-fe93b3a51a90a521b323636677b5f30dab8cfc5baee20a913592187161605cacR488 ? Makes it simpler to read and easier to maintain

Reply:
Ohh. That would be difficult. I'll leave this thread open if anyone would like to suggest such a calculation

Reply:
The comment is also confusing. what are the colons and what are variables and descriptions? Please rewrite